parasol/solr/schema.py

```diff
@@ -0,0 +1,110 @@
+from urllib.parse import urljoin
+from attrdict import AttrDict
+
+class Schema:
+    '''Class for managing Solr schema API.'''
+    def __init__(self, client):
+
+        self.client = client
+        self.url = self.client.build_url(client.schema_handler)
+        self.headers = {
+            'Content-type': 'application/json'
```

Comment:
Does this combine with session headers or replace?

Reply:
It combines. I am going to write a test to be 100% sure about that though.